src/olympia/devhub/models.py

```diff
@@ -25,11 +25,12 @@
 from olympia.users.models import UserProfile
 from olympia.versions.models import Version
 
+
 log = commonware.log.getLogger('devhub')
 
 
 class RssKey(models.Model):
-    key = UUIDField(db_column='rsskey', auto=True, unique=True)
+    key = models.UUIDField(db_column='rsskey', default=uuid.uuid4, unique=True)
```

Comment:
As discussed on IRC replace with a `CharField` with that `default` and a `max_length=36` to match what's in the db right now.